package.json

```diff
@@ -88,7 +88,7 @@
     "eslint-plugin-filenames": "^1.0.0",
     "eslint-plugin-mocha": "^4.5.0",
     "eslint-plugin-one-variable-per-var": "0.0.3",
-    "http-proxy": "^1.15.2",
+    "http-proxy": "^1.18.1",
```

Comment:
Odd that some of these dependencies would change. It _should_ be fine although it increases the risk of unintended consequences.